docs/google/engine.md

```diff
@@ -189,8 +199,15 @@ for b in range(num_circuits_in_batch):
      print(f'Batch #{b}, Sweep #{s}')
      print(results[idx].histogram(key='m'))
      idx+=1
-```
 
+# Alternate way of getting results.
```

Comment:
Nit: consider the less-colloquial "Alternative" rather than "Alternate", which loses the intended meaning in most locales.

Reply:
Is "Alternate" colloquial?  Anyway, changed.